remoting/src/main/java/org/apache/rocketmq/remoting/netty/NettyRemotingAbstract.java

```diff
@@ -88,6 +89,11 @@
     protected Pair<NettyRequestProcessor, ExecutorService> defaultRequestProcessor;
 
     /**
+     * SSL Context.
```

Comment:
No need to use the same text, we'd better to remove this javadoc if we could not further describe this class :-)